test/test_helper.exs

```diff
@@ -21,4 +21,13 @@ defmodule Phoenix.ErrorView do
   end
 end
 
-ExUnit.start()
+:erlang.process_flag(:trap_exit, true)
```

Comment:
Not sure, when I try to connect using `eredis:start_link` or `eredis_client:start_link` without trapping exits I got:

```
** (EXIT from #PID<0.147.0>) {:connection_error, {:connection_error, :econnrefused}}
```